rocAL/source/loaders/image/image_read_and_decode.cpp

```diff
@@ -380,6 +380,16 @@ ImageReadAndDecode::load(unsigned char *buff,
                 _original_width[i] = original_width;
                 _actual_decoded_width[i] = decoded_width;
                 _actual_decoded_height[i] = decoded_height;
+
+                if (_rocjpeg_decoder->is_partial_decoder()) {
```

Comment:
I also think, is_cropped_decoder is better name than is_partial_decoder (please change when changing decoder enums)